CitraRNG/manager_xy.py

```diff
@@ -22,8 +22,11 @@ def getOffsets(self):
 
         self.saveVariable = 0x8C6A6A4
 
+        self.stepCounter = 0
```

Comment:
Looks like a missing offset

Reply:
> Looks like a missing offset

Step counter is only used for DexNav so it's useless for XY. What would be a better way to deal with this?